nightfall-client/src/services/commitment-storage.mjs

```diff
@@ -753,14 +755,61 @@ export async function findUsableCommitmentsMutex(
   );
 }
 
+/**
+ *
+ * @function saveCommitments save a list of commitments in the database
```

Comment:
I suggest to rename the function as e.g. `insertCommitmentsAndResync` (?)

Reply:
Can be. Changed.